docs/install/installing-istio.md

```diff
@@ -13,9 +13,6 @@ installation. If your cloud platform offers a managed Istio installation, the
 [install guide](./README.md) for your specific platform will have those
 instructions.
 
-For example, the [GKE Install Guide](./Knative-with-GKE.md) includes the
```

Comment:
I have removed this as the specific installation like GKE https://github.com/knative/docs/commit/dcaf0b801cda65f1d8ecec736cf479b60eaa7ebb is already removed.